cocotb/share/makefiles/Makefile.inc

```diff
@@ -94,4 +94,12 @@ ifeq ($(OS),Msys)
 
     # For tcl scripts the path to dll has to be in form of C:/User/..
     LIB_DIR := $(shell cygpath -m '$(LIB_DIR)')
+
+    # For Tcl script purposes, generate source file paths in the form of C:/User/... (Icarus does not mind any)
+    ifneq ($(VERILOG_SOURCES),)
+        VERILOG_SOURCES := $(shell cygpath -m $(VERILOG_SOURCES))
+    endif
+    ifneq ($(VHDL_SOURCES),)
+        VHDL_SOURCES := $(shell cygpath -m $(VHDL_SOURCES))
+    endif
```

Comment:
I worry this is going to result in the windows paths being used as make dependencies. Can we move this (and perhaps `LIB_DIR` closer to the usage?

Reply:
Something like now?